libs/input-mapping/tests/Functional/DownloadFilesBranchTest.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\InputMapping\Tests\Functional;
+
+use Keboola\InputMapping\Configuration\File\Manifest\Adapter;
+use Keboola\InputMapping\Exception\InvalidInputException;
+use Keboola\InputMapping\Reader;
+use Keboola\InputMapping\Staging\AbstractStrategyFactory;
+use Keboola\InputMapping\State\InputFileStateList;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\Options\FileUploadOptions;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\StorageApiBranch\Factory\ClientOptions;
+use Psr\Log\NullLogger;
+use Psr\Log\Test\TestLogger;
+
+class DownloadFilesBranchTest extends DownloadFilesTestAbstract
```

Comment:
Tyhle testy nepatří k odstranění c- ne? Chtělo by to alespoň hodit komentář.

Reply:
Je to přesunuté odsud https://github.com/keboola/platform-libraries/pull/82/files#diff-6073149ab5243e1e7c0924cf950ec551d93fb23ebf95e792f88ca485cad2f9fbL271